.eslintrc.js

```diff
@@ -7,18 +7,15 @@ const config = {
         project: ['./tsconfig.lib.json', './tsconfig.test.json'],
     },
     plugins: ['@typescript-eslint', 'import'],
-    extends: [
-        'eslint:recommended',
-        'plugin:@typescript-eslint/recommended',
-        'plugin:@typescript-eslint/recommended-requiring-type-checking',
-        'prettier',
-    ],
+    extends: ['eslint:recommended', 'plugin:@typescript-eslint/strict-type-checked', 'prettier'],
```

Comment:
Some rules were dropped from `@typescript-eslint/recommended`, so "upgraded" to `typescript-eslint/strict-type-checked`.